src/styles/quiz-card/quiz-card.css

```diff
@@ -0,0 +1,3 @@
+h2 {
```

Comment:
This should be a class:
```suggestion
.quiz-card__question {
```

Reply:
Thanks, h2 was just a placeholder so I could commit my files without husky interferring. Created a class.